.github/workflows/CI.yml

```diff
@@ -70,7 +70,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: julia-actions/setup-julia@v1
         with:
-          version: '1'
+          version: '1.6'
```

Comment:
Want to pin to a specific minor version so that formatting changes don't break things for us